kano_profile/tracker.py

```diff
@@ -166,7 +166,7 @@ def session_end(session_file):
                 with wf:
                     json.dump(data, wf)
                 if 'SUDO_USER' in os.environ:
-                    chown_path(data)
+                    chown_path(session_file)
```

Comment:
Is it not a problem trying to set the owner with the file already opened, is there not potential for a later write overwriting these permissions?